client-templates/github/accept.json.sample

```diff
@@ -68,10 +68,22 @@
             "value": ".snyk"
           }
         ]
+      },
+    {
+        "//": "used for pushing up webhooks from github",
+        "method": "POST",
+        "path": "/webhook/github/:webhookId"
       }
   ],
   "private":
   [
+    {
+      "//": "list the user's info",
+      "method": "GET",
+      "path": "/user",
+      "origin": "https://${GITHUB_TOKEN}@${GITHUB_API}"
+    },
```

Comment:
What do we need the user info for in our agent flow? Creds validation?

Reply:
exactly.